modules/microshift-low-latency-config-yaml.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-config-yaml_{context}"]
+= Configuration kubelet parameters and values in {microshift-short}
+
+The first step in enabling low latency to a {microshift-short} cluster is to add configurations to the {microshift-short} `config.yaml` file.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have root access to the cluster.
+* You made a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory, renaming it `config.yaml`.
+
+.Procedure
+
+. Add the kubelet configuration to the {microshift-short} `config.yaml` file:
++
+.Example passthrough `kubelet` configuration
+[source,yaml]
+----
+# ...
```

Comment:
The OCP core docs [require the metadata in YAML code blocks](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#required-fields). Does the same apply for MicroShift?

Reply:
We haven't done this because we have so many of these, but fixing the Config book is on our list. And at once point, another reviewer told me we didn't always have to do this in certain cases (I'll have to find that comment again to tell you what  the reason was). Meanwhile, I'll leave this one, but created an issue to clarify for this docset. https://issues.redhat.com/browse/OSDOCS-11859